web/pandas/pdeps/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Those Makefile and make.bat files are not actually needed, but we could keep them in case you locally just want to build the PDEPs (although then you can also run `sphinx-build -M html . _build`, so that might not be worth it)